### PR TITLE
Schedule test

### PR DESCRIPTION
**Pull request by AsterNighT**
### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Breaking backward compatibility

Related changes

- [ ] Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: api/v1alpha1/schedule_type_test.go [hunk 1]
```diff
@@ -0,0 +1,88 @@
+// Copyright 2019 Chaos Mesh Authors.
```
Comment (fewdan) [inline api/v1alpha1/schedule_type_test.go:1]: s/2019/2021/g  ?

**Review by fewdan — CHANGES_REQUESTED** [review]

File: api/v1alpha1/schedule_type_test.go (hunk 1 above)
Comment (AsterNighT) [reply api/v1alpha1/schedule_type_test.go:1]: oops, that is copy&pasted. I may do a global fix on this.

**Comment by AsterNighT** [comment]
PTAL @fewdan @YangKeao

**Review by YangKeao — APPROVED** [review]
LGTM

**Pull request merged**